decidim-core/app/jobs/decidim/machine_translation_create_resource_job.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Decidim
+  class MachineTranslationCreateResourceJob < ApplicationJob
+    queue_as :default
+
+    def perform(resource, source_locale)
+      class_name = resource.class.name
+      id = resource.id
+      translatable_fields = resource.class.translatable_fields_list.map(&:to_s)
+      translatable_fields.each do |field|
+        locales = Decidim.available_locales.map(&:to_s)
+        locales.each do |locale|
+          next if locale == source_locale
+          next if resource_field_value(resource, field, source_locale).blank?
+
+          MachineTranslationCreateFieldsJob.perform_later(
+            id,
```

Comment:
Can't we pass the resource as an object to the other job? The same way we do it here.